src/api/internal.rs

```diff
@@ -658,6 +658,7 @@ impl<T: Pixel> ContextInner<T> {
 
       let mvs = &fs.frame_mvs[index];
       use byteorder::{NativeEndian, WriteBytesExt};
+      // dynamic allocation: once per frame
```

Comment:
maybe worth noting this is from dump_lookahead_data?

Reply:
Ah, missed that.